ReviewMonitor/AppHelper.swift

```diff
@@ -8,10 +8,60 @@
 
 import UIKit
 
+import SystemConfiguration
+
 enum DateFormat: String {
     case MMMddyyy = "MMM dd, yyyy"
 }
 
+class ReachabilityManager: NSObject {
```

Comment:
Instead of creating a class here, we should create a new file called `ReachabilityManager`.